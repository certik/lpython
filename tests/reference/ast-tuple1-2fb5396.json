{
    "basename": "ast-tuple1-2fb5396",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/tuple1.py",
    "infile_hash": "976ce5ef13850c83aceb79156dc1e8d30489d695252188d6ffe26809",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-tuple1-2fb5396.stdout",
    "stdout_hash": "a4dd50845f4f2d065041a32c7d1c306b44825234b3f8088a38918313",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}