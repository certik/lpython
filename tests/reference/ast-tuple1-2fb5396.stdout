(Module [(FunctionDef test_Tuple ([] [] [] [] [] [] []) [(AnnAssign (Name a1 Store) (Subscript (Name tuple Load) (Tuple [(Name i32 Load) (Name i32 Load) (Name i32 Load)] Load) Load) () 1) (Assign [(Name a1 Store)] (Tuple [(ConstantInt 1 ()) (ConstantInt 2 ()) (ConstantInt 3 ())] Load) ()) (Assign [(Name a1 Store)] (Tuple [(UnaryOp USub (ConstantInt 3 ())) (UnaryOp USub (ConstantInt 2 ())) (UnaryOp USub (ConstantInt 1 ()))] Load) ()) (AnnAssign (Name a2 Store) (Subscript (Name tuple Load) (Tuple [(Name str Load) (Name str Load) (Name str Load)] Load) Load) () 1) (Assign [(Name a2 Store)] (Tuple [(ConstantStr "a" ()) (ConstantStr "b" ()) (ConstantStr "c" ())] Load) ()) (AnnAssign (Name a3 Store) (Subscript (Name tuple Load) (Tuple [(Name i32 Load) (Name i32 Load) (Name f32 Load) (Name str Load)] Load) Load) () 1) (Assign [(Name a3 Store)] (Tuple [(UnaryOp USub (ConstantInt 2 ())) (UnaryOp USub (ConstantInt 1 ())) (ConstantFloat 0.450000 ()) (ConstantStr "d" ())] Load) ()) (AnnAssign (Name a4 Store) (Subscript (Name tuple Load) (Tuple [(Subscript (Name tuple Load) (Tuple [(Name i32 Load) (Name i32 Load) (Name i32 Load)] Load) Load) (Subscript (Name tuple Load) (Tuple [(Name i32 Load) (Name i32 Load) (Name i32 Load)] Load) Load)] Load) Load) () 1) (Assign [(Name a4 Store)] (Tuple [(Tuple [(ConstantInt 1 ()) (ConstantInt 2 ()) (ConstantInt 3 ())] Load) (Tuple [(ConstantInt 4 ()) (ConstantInt 5 ()) (ConstantInt 6 ())] Load)] Load) ()) (AnnAssign (Name a5 Store) (Subscript (Name tuple Load) (Tuple [(Subscript (Name tuple Load) (Tuple [(Name str Load) (Name str Load) (Name f32 Load)] Load) Load) (Subscript (Name tuple Load) (Tuple [(Name str Load) (Name i32 Load) (Name f32 Load)] Load) Load)] Load) Load) () 1) (Assign [(Name a5 Store)] (Tuple [(Tuple [(ConstantStr "a" ()) (ConstantStr "b" ()) (ConstantFloat 3.400000 ())] Load) (Tuple [(ConstantStr "c" ()) (ConstantInt 3 ()) (ConstantFloat 5.600000 ())] Load)] Load) ())] [] () ())] [])
